test/tests/ver-1ha/comparison_ver-1ha.py

```diff
@@ -0,0 +1,136 @@
+import csv
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+from scipy import special
+import os
+import math
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+# ===============================================================================
+# Extract TMAP8 results
+tmap8_sol = pd.read_csv("./gold/ver-1ha_out.csv")
+tmap8_sol_time = tmap8_sol['time']
+tmap8_sol_P2 = tmap8_sol['P2_val']
+tmap8_sol_P3 = tmap8_sol['P3_val']
+tmap8_sol_C2 = tmap8_sol['C2_val']
+tmap8_sol_C3 = tmap8_sol['C3_val']
+
+
+# ===============================================================================
+# Calculate analytical solution
+def get_analytical_solution(t_vect):
+    """Returns the pressure and concentration for the 2nd and 3d enclosures as a function of time
+    Args:
+        t_vect (ndarray): time in s
+    Returns:
+        P2 (np.array): pressure of T3 in the 2nd enclosure in Pa
+        P3 (np.array): pressure of T3 in the 3rd enclosure in Pa
+        C2 (np.array): concentration of T3 in the 2nd enclosure in atoms/m^3
+        C3 (np.array): concentration of T3 in the 3rd enclosure in atoms/m^3
+    """
+    P1 = 1.0    # Pa
+    Q = 0.1    # m^3/s
+    V2 = 1      # m^3
+    V3 = 1      # m^3
+    R = 8.31446261815324  # J/K/mol
+    T = 303  # K
+    N_a = 6.02214076e23  # at/mol
+
+    P2 = P1*(1-np.exp(-Q*tmap8_sol_time/V2))
+    if (V2 == V3):
+        P3 = P1*(1 - (1 + Q*tmap8_sol_time/V2)*np.exp(-Q*tmap8_sol_time/V2))
+    else:
+        P3 = P1*(1 - (V2/(V2-V3))*np.exp(-Q*tmap8_sol_time/V2) +
+                 (V2/(V2-V3))*np.exp(-Q*tmap8_sol_time/V3))
+
+    # Convert pressures (Pa) to concentrations (atoms/m^3)
+    # following ideal gas law
+    C2 = P2*N_a/(R*T)
+    C3 = P3*N_a/(R*T)
+    return (P2, P3, C2, C3)
+
+
+P2, P3, C2, C3 = get_analytical_solution(
+    tmap8_sol_time)
```

Comment:
```suggestion
P2, P3, C2, C3 = get_analytical_solution(tmap8_sol_time)
```

Reply:
This was formatted using the Python linter.